#request fields
bool enable
---
#response fields
string res
